package.json

```diff
@@ -6,7 +6,9 @@
   "type": "module",
   "scripts": {
     "start:indexer": "node indexer/index.js",
+    "start:indexer:heapstats": "node indexer/index.js --heap-stats",
     "start:indexer:prod": "pm2 start yarn --interpreter bash --name nina-indexer -- start:indexer",
+    "start:indexer:prod:heapstats": "pm2 start yarn --interpreter bash --name nina-indexer -- start:indexer --heap-stats",
```

Comment:
this line could use the `start:indexer:heapstats` script defined above, so:
`"start:indexer:prod:heapstats": "pm2 start yarn --interpreter bash --name nina-indexer -- start:indexer:heapstats"`

Reply:
Added.